api/app/GraphQL/Mutations/CreateOrUpdateScreeningQuestionAssessmentStep.php

```diff
@@ -43,7 +58,8 @@ public function __invoke($_, array $args)
 
                     $questionToUpdate->save();
                 } else {
-                    $pool->generalQuestions()->Create([
+                    $pool->screeningQuestions()->create([
+                        'assessment_step_id' => $assessmentStep->id,
```

Comment:
This doesn't work now with assessment_step_id taken out of the `fillable` array. You'll need to create a ScreeningQuestions object, set assessment_step_id, then save it to the pool.

Reply:
Mass assignment sure is convenient. 
I had to make the fields nullable so the process would work. Then added a test afterwards for it. 
Assertion `assertSuccessful()` incorrectly returns true when the mutation fails off mass assignment which tripped me up for a bit, so I tested cautiously around that. 

https://github.com/GCTC-NTGC/gc-digital-talent/pull/9161/commits/6c4e2667f7b77bb1eead3b827e0e5e470632e9fc
https://github.com/GCTC-NTGC/gc-digital-talent/pull/9161/commits/69ec9749b4c9f0279534be1808a3204eee5a2914